thirdparty/luasocket/CMakeLists.txt

```diff
@@ -17,6 +17,11 @@ list(APPEND PATCH_CMD COMMAND sh -c "\"\$@\" src/*" --
 # Don't enforce CFLAGS.
 list(APPEND PATCH_CMD COMMAND ${ISED} "s|-O2 -ggdb3 ||g" src/makefile)
 
+# Make sure socket fds are CLOEXEC on *nix
```

Comment:
if(NOT WIN32)

Reply:
Given your remark in https://github.com/koreader/koreader-base/pull/1827#issuecomment-2174289775 ;-)

Reply:
It's already patching a file that's never ever even built on windows, so it's perfectly harmless ;).